src/desktop/apps/isomorphic-relay-example/components/Layout.js

```diff
@@ -0,0 +1,37 @@
+import Nav from './Nav'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { ContextProvider } from '@artsy/reaction/dist/Components/Artsy'
+import { data as sd } from 'sharify'
```

Comment:
This file is used for SSR too, right? In which case this singleton access is unsafe, it needs to use per-request data.

Reply:
Now that we have TS in Force, we should probably import the types I had for `sharify` in Reaction, which only exposed the per-request data when accessed through `req.locals.sd`

Reply:
Nice catch!